Packs/Akamai_SIEM/README.md

```diff
@@ -0,0 +1,43 @@
+<~XSIAM>
+ 
+## Overview
+A Web Application Firewall (WAF) is a security filter that protects against HTTP-based attacks by inspecting traffic before it reaches your application.
+ 
+## This pack includes:
+ 
+Data normalization capabilities:
+  * Rules for Parsing and Modeling logs that are ingested via the Native Collector on Cortex XSIAM.
+  * The ingested Akamai WAF SIEM logs can be queried in XQL Search using the *`akamai_waf_raw`* dataset.
```

Comment:
* The `akamai_waf_raw` dataset enables querying ingested Akamai WAF SIEM logs in XQL Search.